sphinx/application.py

```diff
@@ -26,6 +26,7 @@
 from docutils import nodes
 from docutils.parsers.rst import convert_directive_function, \
     directives, roles
+from pkg_resources import iter_entry_points
```

Comment:
Right. And `setuptools` needs to be added to `install_requires` in `setup.py` if it is to be a non-optional dependency.